src/monero_dispatch.c

```diff
@@ -115,13 +100,13 @@ int check_ins_access() {
         case INS_MLSAG:
         case INS_CLSAG:
         case INS_GEN_COMMITMENT_MASK:
-            if (os_global_pin_is_validated() != PIN_VERIFIED) {
-                return SW_SECURITY_PIN_LOCKED;
-            }
             if (G_monero_vstate.tx_in_progress != 1) {
                 return SW_COMMAND_NOT_ALLOWED;
             }
             return SW_OK;
+        case INS_LOCK_DISPLAY:
+            // Deprecated command
+            return SW_INS_NOT_SUPPORTED;
```

Comment:
Do you known if many software wallets use this?

Reply:
I had a look at the official wallet : Monero GUI which doesnt use it, and also had a look at Feather wallet and Monerujo that don't use it either :)